code/modules/admin/verbs/randomverbs.dm

```diff
@@ -837,10 +837,15 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!holder)
 		return
 
+	var/view = src.view
 	if(view == world.view)
 		view = input("Select view range:", "FUCK YE", 7) in list(1,2,3,4,5,6,7,8,9,10,11,12,13,14,128)
 	else
 		view = world.view
+	if(mob)
+		mob.set_viewsize(view)
+	else

```

Comment:
`mob` can never be unset. Attempting to set it to null does nothing. It's protected by byond, so no need to test.